### PR TITLE
Revert TGUI Vore Panel

### LINKED ISSUE #11436: Map votes still broken
## Test Merges: 
Not Supplied / None
## Reproduction:
Map votes are still broken, cannot be started by non admins. The issue is Bibby's PR that changed þe UI
 Submitted by: Tipy (294147033848217600)

### PR DESCRIPTION
**Pull request by UselessTheremin**
### Intent of your Pull Request

fucking bibby broke map votes, i enjoy that we've been playing non-box maps fore a change but it's a bug. and i guess i should fix those or something. well, the easiest fix is to remove it since bibby obviously isnt gonna fix it. if someone steps up to fix the vote panel, i'll close this pr, but until then it's a pretty urgent issue.

Reverts #11170
Closes #11436

### Why is this change good for the game?

it's not, lesser of two evils

:cl:    
rscdel: removed tgui vote panel
/:cl:

### CODE REVIEW

**Comment by Bibby0110** [comment]
Improve don't remove

**Comment by Bibby0110** [comment]
Also don't re-line the wasp file and don't re-add the distributable js chunks

**Comment by UselessTheremin** [comment]
> Also don't re-line the wasp file and don't re-add the distributable js chunks

i just reverted the commit i have no fucken clue how to do tgui code but its broken, can you at least fix it mate

**Comment by UselessTheremin** [comment]
or at least revert it yourself

**Comment by monster860** [comment]
never trust code from a coder that isn't a femboy

**Comment by Bibby0110** [comment]
> never trust code from a coder that isn't a femboy

Is this why monstermos doesn’t work?

**Comment by MegaEmpirical** [comment]
"Improve don't remove" only works if someone actually does so.

**Comment by Bibby0110** [comment]
> "Improve don't remove" only works if someone actually does so.

No, if you're capable of removing it you're capable of improving it

**Comment by UselessTheremin** [comment]
> > "Improve don't remove" only works if someone actually does so.
> 
> No, if you're capable of removing it you're capable of improving it

i literally dont know how to code with tgui, do you want me to just learn an entire other type of code because what, the pr you made doesnt fucken work? fix you own pr for the love of god, if you do im closing this

**Review by JamieD1 — APPROVED** [review]

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/wasp.dm [hunk 1]
```diff
@@ -1,85 +1,85 @@
-//A wasp that charges at their enemy, injecting them with poison
-/mob/living/simple_animal/hostile/asteroid/wasp
```
Comment (Bibby0110) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/wasp.dm:2]: no touchy this file

File: tgui/public/tgui-common.chunk.js [hunk 2]
```diff
@@ -0,0 +1 @@
...ns){var n=re.defaults.walkTokens;t.walkTokens=function(t){e.walkTokens(t),n&&n(t)}}re.setOptions(t)},re.walkTokens=function(e,t){for(var r,o=n(e);!(r=o()).done;){var i=r.value;switch(t(i),i.type){case"table":for(var u,a=n(i.tokens.header);!(u=a()).done;){var c=u.value;re.walkTokens(c,t)}for(var s,l=n(i.tokens.cells);!(s=l()).done;)for(var f,d=n(s.value);!(f=d()).done;){var p=f.value;re.walkTokens(p,t)}break;case"list":re.walkTokens(i.items,t);break;default:i.tokens&&re.walkTokens(i.tokens,t)}}},re.parseInline=function(e,t){if(null==e)throw new Error("marked.parseInline(): input parameter is undefined or null");if("string"!=typeof e)throw new Error("marked.parseInline(): input parameter is of type "+Object.prototype.toString.call(e)+", string expected");t=Z({},re.defaults,t||{}),Q(t);try{var n=K.lexInline(e,t);return t.walkTokens&&re.walkTokens(n,t.walkTokens),X.parseInline(n,t)}catch(r){if(r.message+="\nPlease report this to https://github.com/markedjs/marked.",t.silent)return"<p>An error occurred:</p><pre>"+J(r.message+"",!0)+"</pre>";throw r}},re.Parser=X,re.parser=X.parse,re.Renderer=$,re.TextRenderer=H,re.Lexer=K,re.lexer=K.lex,re.Tokenizer=S,re.Slugger=W,re.parse=re,re}()}]]);
```
Comment (Bibby0110) [inline tgui/public/tgui-common.chunk.js:1]: no touchy

File: tgui/public/tgui-panel.bundle.css [hunk 3]
```diff
@@ -0,0 +1 @@
...or:#410194}50%{color:#7f17d8}75%{color:#410194}to{color:#3bb5d3}}.theme-light .phobia{color:#d00;font-weight:700;animation:phobia .75s infinite}@keyframes phobia{0%{color:#0d0d0d}50%{color:#d00}to{color:#0d0d0d}}.theme-light .icon{height:1em;width:auto}.theme-light .memo{color:#638500;text-align:center}.theme-light .memoedit{text-align:center;font-size:125%}.theme-light .abductor{color:purple;font-style:italic}.theme-light .mind_control{color:#a00d6f;font-size:100%;font-weight:700;font-style:italic}.theme-light .slime{color:#00ced1}.theme-light .drone{color:#848482}.theme-light .monkey{color:#975032}.theme-light .swarmer{color:#2c75ff}.theme-light .resonate{color:#298f85}.theme-light .monkeyhive{color:#774704}.theme-light .monkeylead{color:#774704;font-size:80%}.theme-light .connectionClosed,.theme-light .fatalError{background:red;color:#fff;padding:5px}.theme-light .connectionClosed.restored{background:green}.theme-light .internal.boldnshit{color:#00f;font-weight:700}.theme-light .text-normal{font-weight:400;font-style:normal}.theme-light .hidden{display:none;visibility:hidden}.theme-light .ml-1{margin-left:1em}.theme-light .ml-2{margin-left:2em}.theme-light .ml-3{margin-left:3em}
```
Comment (Bibby0110) [inline tgui/public/tgui-panel.bundle.css:1]: no touchy

File: tgui/public/tgui-panel.bundle.js [hunk 4]
```diff
@@ -0,0 +1 @@
...e)(2,r.Flex,{align:"center",className:"Notification",children:[(0,o.createComponentVNode)(2,r.Flex.Item,{className:"Notification__content",grow:1,children:n}),t&&(0,o.createComponentVNode)(2,r.Flex.Item,{className:"Notification__rightSlot",children:t})]})}},80:function(e,t,n){"use strict";t.__esModule=!0,t.saveChatToDisk=t.changeScrollTracking=t.removeChatPage=t.toggleAcceptedType=t.updateChatPage=t.changeChatPage=t.addChatPage=t.updateMessageCount=t.rebuildChat=t.loadChat=void 0;var o=n(22),r=n(106),a=(0,o.createAction)("chat/load");t.loadChat=a;var i=(0,o.createAction)("chat/rebuild");t.rebuildChat=i;var c=(0,o.createAction)("chat/updateMessageCount");t.updateMessageCount=c;var s=(0,o.createAction)("chat/addPage",(function(){return{payload:(0,r.createPage)()}}));t.addChatPage=s;var d=(0,o.createAction)("chat/changePage");t.changeChatPage=d;var l=(0,o.createAction)("chat/updatePage");t.updateChatPage=l;var u=(0,o.createAction)("chat/toggleAcceptedType");t.toggleAcceptedType=u;var g=(0,o.createAction)("chat/removePage");t.removeChatPage=g;var p=(0,o.createAction)("chat/changeScrollTracking");t.changeScrollTracking=p;var h=(0,o.createAction)("chat/saveToDisk");t.saveChatToDisk=h}});
```
Comment (Bibby0110) [inline tgui/public/tgui-panel.bundle.js:1]: no touchy

File: tgui/public/tgui.bundle.css [hunk 5]
```diff
@@ -0,0 +1 @@
...A4LjQ4LS4xMjkgOC40OS0uMDAyLjQxNyA2LjQxNS0xLjc1NCA5LjQ1My00LjEyNCAxMi41NjEtMi40MTcgMy4xNy01LjE0NSA2Ljc5LTQuMDAzIDEzLjAwMyAxLjUwOCA4LjIwMyAxMC4xODQgMTAuNTk3IDE0LjYyMiA5LjMxMi0zLjMxOC0uNS01LjMxOC0xLjc1LTUuMzE4LTEuNzVzMS44NzYuOTk5IDUuNjUtMS4zNmMtMy4yNzYuOTU2LTEwLjcwNC0uNzk3LTExLjgtNi43NjMtLjk1OC01LjIwOC45NDYtNy4yOTUgMy40LTEwLjUxNCAyLjQ1NS0zLjIyIDUuMjg1LTYuOTU5IDQuNjg1LTE0LjQ4OWwuMDAzLjAwMmg4LjkyN3YtNTZjLTE1LjA3Mi0zLjg3MS0yNy42NTMtNi4zNi0zNy43NDctNy40NjVDMTE0LjI3OS41NTIgMTA0LjA0NiAwIDkzLjUzNyAwem03MC4zMjEgMTcuMzA5bC4yMzggNDAuMzA1YzEuMzE4IDEuMjI2IDIuNDQgMi4yNzggMy4zNDEgMy4xMDYgNC44NCA2Ljc3NSA4Ljg1IDE2LjI0NiAxMi4wMyAyOC40MTRIMjAwdi01NmMtNi42NzctNC41OTQtMTkuODM2LTEwLjQ3My0zNi4xNC0xNS44MjV6bS0yOC4xMiA1LjYwNWw4LjU2NSAxNy43MTdjLTExLjk3LTYuNDY3LTEzLjg0Ny05LjcxNy04LjU2NS0xNy43MTd6bTIyLjc5NyAwYzIuNzcxIDggMS43ODcgMTEuMjUtNC40OTQgMTcuNzE3bDQuNDk0LTE3LjcxN3ptMTUuMjIyIDI0LjAwOWw4LjU2NSAxNy43MTZjLTExLjk3LTYuNDY2LTEzLjg0Ny05LjcxNy04LjU2NS0xNy43MTZ6bTIyLjc5NyAwYzIuNzcxIDggMS43ODcgMTEuMjUtNC40OTQgMTcuNzE2bDQuNDk0LTE3LjcxNnpNOTcuNDQgNDkuMTNsOC41NjUgMTcuNzE2Yy0xMS45Ny02LjQ2Ny0xMy44NDctOS43MTctOC41NjUtMTcuNzE2em0yMi43OTUgMGMyLjc3MiA3Ljk5OSAxLjc4OCAxMS4yNS00LjQ5MyAxNy43MTZsNC40OTMtMTcuNzE2eiIvPjwvc3ZnPg==)}
```
Comment (Bibby0110) [inline tgui/public/tgui.bundle.css:1]: no touchy

File: tgui/public/tgui.bundle.js [hunk 6]
```diff
@@ -0,0 +1 @@
...createComponentVNode)(2,i.Window,{width:400,height:600,children:(0,o.createComponentVNode)(2,i.Window.Content,{children:!c&&(0,o.createComponentVNode)(2,a.NoticeBox,{children:"No question is currently being asked."})||(0,o.createComponentVNode)(2,l)})})};var l=function(e,t){var n=(0,r.useBackend)(t),i=n.act,l=n.data,d=l.question,u=l.answers,s=void 0===u?[]:u;return(0,o.createComponentVNode)(2,a.Section,{children:[(0,o.createComponentVNode)(2,a.Box,{bold:!0,textAlign:"center",fontSize:"16px",m:1,children:['"',d,'"']}),(0,o.createComponentVNode)(2,a.Grid,{children:s.map((function(e){return(0,o.createComponentVNode)(2,a.Grid.Column,{children:[(0,o.createComponentVNode)(2,a.Button,{fluid:!0,bold:!0,content:(0,c.toTitleCase)(e.answer),selected:e.selected,fontSize:"16px",lineHeight:"24px",textAlign:"center",mb:1,onClick:function(){return i("vote",{answer:e.answer})}}),(0,o.createComponentVNode)(2,a.Box,{bold:!0,textAlign:"center",fontSize:"30px",children:e.amount})]},e.answer)}))})]})}},function(e,t,n){"use strict";t.__esModule=!0,t.Electrolyzer=void 0;var o=n(0),r=n(2),a=n(1),c=n(3);t.Electrolyzer=function(e,t){var n=(0,r.useBackend)(t),i=n.act,l=n.data;return(0,o.createComponentVNode)(
```
Comment (Bibby0110) [inline tgui/public/tgui.bundle.js]: no touchy

**Review by Bibby0110 — CHANGES_REQUESTED** [review]
no touchy jamie

**Comment by UselessTheremin** [comment]
whatever this pr is being fixed by theos who is more competent than i am, i'll save my spaghetti for the mothers day dinner

**Pull request closed without merging**